Reportedly/Designs/NavigationController.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  NavigationController.swift
+//  Reportedly
+//
+//  Created by Mikey Pham on 11/13/20.
+//  Copyright © 2020 NimbleHQ. All rights reserved.
+//
+
+import UIKit
+
+final class NavigationController: UINavigationController {
+    
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        return .default
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setUpNavigationBar()
+        setColors()
```

Comment:
I just want to notice that in the1 we were trying to use `setUp` instead of `set`. 😅

Reply:
got it, let me update the naming for entire app, I think this `setColors` is used in multiple places 🙏